tests/Resources/RHOSi.resource

```diff
@@ -40,3 +41,8 @@ Protect Sensitive Variables In Keywords
     [Documentation]    Register keywords which use sensitive data as "Protected"
     ...                to turn their log level to NONE using RobotLogListener by Robocorp
     Register Protected Keywords    names=@{PROTECTED_KEYWORDS}
+
+Initialize Global Variables
+    [Documentation]   Set varibled globally
+    ${RHODS_VERSION}    Get RHODS Version
```

Comment:
From the style guide: _We’ll use the equal sign for variable assignments without having a space after the variable name (e.g. ${dashboard_url}=  Get Dashboard URL)_

Reply:
done